pom.xml

```diff
@@ -63,7 +82,7 @@
                             <archive>
                                 <manifest>
                                     <mainClass>
-                                        by.mariayuran.homework.TicketService
+                                        by.mariayuran.by.mariayuran.homework.TicketService
```

Comment:
its correct?

Reply:
no, how is it?

Reply:
So just fix it. The IDE may have done something wrong)